sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1677,8 +1681,9 @@ object GpuOverrides {
     expr[Min](
       "Min aggregate operator",
       ExprChecks.fullAgg(
-        TypeSig.commonCudfTypes + TypeSig.NULL, TypeSig.orderable,
-        Seq(ParamCheck("input", TypeSig.commonCudfTypes + TypeSig.NULL, TypeSig.orderable))),
+        TypeSig.commonCudfTypes + TypeSig.DECIMAL + TypeSig.NULL, TypeSig.orderable,
```

Comment:
If we are going to add DECIMAL for MIN in all cases we need some tests to verify that it will work for groupby and reduction use cases as well.